locales/en.yml

```diff
@@ -19,6 +20,9 @@ en:
           comment:
             syntax: jira comment on <issue> <comment text>
             desc: Adds <comment text> to <issue>
+          point:
```

Comment:
There seems to be a mismatch here between the translation namespace defined in this file (`point`) and the one you're using to reference it in the handler code (`points`)